cypress/integration/v2/blocks/card.spec.js

```diff
@@ -58,3 +63,403 @@ function switchDesign() {
+
+	cy.collapse( 'Container' )
+
+	desktopOnly( () => {
+		cy.adjust( 'Background', {
+			[ `Color Type` ]: 'single',
+			[ `Background Color` ]: '#000000',
+			[ `Background Color Opacity` ]: 0.7,
+		} ).assertComputedStyle( {
+			'.ugb-card__item': {
+				[ `background-color` ]: 'rgba(0, 0, 0, 0.7)',
+			},
+		} )
+	} )
+
+	cy.setBlockAttribute( {
+		[ `column${ viewport === 'Desktop' ? '' : viewport }BackgroundMediaUrl` ]: Cypress.env( 'DUMMY_IMAGE_URL' ),
+		[ `image1Url` ]: Cypress.env( 'DUMMY_IMAGE_URL' ),
+		[ `image2Url` ]: Cypress.env( 'DUMMY_IMAGE_URL' ),
+		[ `image3Url` ]: Cypress.env( 'DUMMY_IMAGE_URL' ),
+	} )
+
+	desktopOnly( () => {
+		cy.adjust( 'Background', {
+			[ `Background Media Tint Strength` ]: 7,
+			[ `Fixed Background` ]: true,
+			[ `Adv. Background Image Settings` ]: {
+				[ `Image Blend Mode` ]: 'hue',
+			},
+		} ).assertComputedStyle( {
+			'.ugb-card__item': {
+				[ `background-image` ]: `url("${ Cypress.env( 'DUMMY_IMAGE_URL' ) }")`,
+				[ `background-attachment` ]: 'fixed',
+				[ `background-blend-mode` ]: 'hue',
+			},
+			'.ugb-card__item:before': {
+				[ `opacity` ]: '0.7',
+			},
+		} )
+	} )
+
+	cy.adjust( 'Background', {
+		[ `Adv. Background Image Settings` ]: {
+			[ `Image Position` ]: {
+				viewport,
+				value: 'center center',
+			},
+			[ `Image Repeat` ]: {
+				viewport,
+				value: 'repeat-x',
+			},
+			[ `Image Size` ]: {
+				viewport,
+				value: 'custom',
+			},
+			[ `Custom Size` ]: {
+				viewport,
+				value: 19,
+				unit: '%',
+			},
+		},
+	} ).assertComputedStyle( {
+		'.ugb-card__item': {
+			[ `background-position` ]: '50% 50%',
+			[ `background-repeat` ]: 'repeat-x',
+			[ `background-size` ]: '19%',
+		},
+	} )
+
+	cy.adjust( 'Background', {
+		[ `Adv. Background Image Settings` ]: {
+			[ `Image Size` ]: {
+				viewport,
+				value: 'custom',
+			},
+			[ `Custom Size` ]: {
+				viewport,
+				value: 23,
+				unit: 'px',
+			},
+		},
+	} ).assertComputedStyle( {
+		'.ugb-card__item': {
+			[ `background-size` ]: '23px',
+		},
+	} )
+
+	cy.adjust( 'Background', {
+		[ `Adv. Background Image Settings` ]: {
+			[ `Image Size` ]: {
+				viewport,
+				value: 'custom',
+			},
+			[ `Custom Size` ]: {
+				viewport,
+				value: 8,
+				unit: 'vw',
+			},
+		},
+	} ).assertComputedStyle( {
+		'.ugb-card__item': {
+			[ `background-size` ]: '8vw',
+		},
+	} )
+
+	desktopOnly( () => {
+		cy.adjust( 'Borders', 'solid' )
+		cy.adjust( 'Border Width', 4 )
+		cy.adjust( 'Border Color', '#a12222' )
+		cy.adjust( 'Border Radius', 26 ).assertComputedStyle( {
+			'.ugb-card__item': {
+				[ `border-style` ]: 'solid',
+				[ `border-top-width` ]: '4px',
+				[ `border-bottom-width` ]: '4px',
+				[ `border-left-width` ]: '4px',
+				[ `border-right-width` ]: '4px',
+				[ `border-color` ]: '#a12222',
+				[ `border-radius` ]: '26px',
+			},
+		} )
+		cy.adjust( 'Shadow / Outline', 7 )
+			.assertClassName( '.ugb-card__item', 'ugb--shadow-7' )
+	} )
+
+	cy.adjust( 'Borders', 'dashed' )
```

Comment:
do we still need this for tablet and mobile assertions?

Reply:
yes, we need this to be able to adjust the border width responsive options